scene/resources/mesh.h

```diff
@@ -159,11 +159,43 @@ class Mesh : public Resource {
 	Size2i get_lightmap_size_hint() const;
 	void clear_cache() const;
 
-	typedef Vector<Vector<Face3>> (*ConvexDecompositionFunc)(const Vector<Face3> &p_faces, int p_max_convex_hulls);
+	struct ConvexDecompositionSettings {
+		enum Mode : int {
+			CONVEX_DECOMPOSITION_MODE_VOXEL = 0,
+			CONVEX_DECOMPOSITION_MODE_TETRAHEDRON
+		};
+
+		/// Maximum concavity. [Range: 0.0 -> 1.0]
+		real_t max_concavity = 1.0;
```

Comment:
Same with concavity, the default in VHACD is 0.001, is there a reason to set it to 1 by default instead? I've tried to change the value and I don't see any difference in generated meshes though.

Reply:
This paramter is quite important, together with `resolution` and `max_convex_hulls`. By setting it to `0.001` you are telling VHACD to consider really little concavity and so introduce another shape.

For example, if you set:
```
max_convex_hulls: 1024
max_concavity: 0.001
```
This will generate a lot of shapes, that will describe the Mesh silhouette really well.

If you set something like:
```
max_convex_hulls: 1
max_concavity: 0.001

// or
max_convex_hulls: 1024
max_concavity: 1
```
It will generate only 1 Collision shape that tries to describe the mesh silhouette as best as it can with just one convex shape.